### PR TITLE
Update security docs

### PR DESCRIPTION
**Pull request by stsewd**
We missed to mention the latest security discoveries in our changelog as we have been doing.

---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--9973.org.readthedocs.build/en/9973/

- Developer's documentation (`dev`): https://dev--9973.org.readthedocs.build/en/9973/

### CODE REVIEW

File: docs/user/security.rst [hunk 1]
```diff
@@ -49,8 +49,9 @@ You can expect:
 * We will respond acknowledging your email typically within one business day.
 * We will follow up if and when we have confirmed the issue with a timetable for the fix.
 * We will notify you when the issue is fixed.
-* We will add the issue to our :ref:`security issue archive <security:Security issue archive>`.
+* We will create a `GitHub advisory`_ and publish it when the issue has been fixed.
```
Comment (benjaoming) [inline docs/user/security.rst:52]:
```suggestion
* We will create a `GitHub advisory`_ and publish it when the issue has been fixed and deployed on all platforms.
```

**Review by benjaoming — APPROVED** [review]
LGTM - just added a small clarification in case it makes sense.

File: docs/user/security.rst (hunk 1 above)
Comment (stsewd) [reply docs/user/security.rst]: Changed it to `our platforms`, otherwise it could give the idea we are notifying custom installations :D

Comment (benjaoming) [reply docs/user/security.rst]: That's a good clarification to the clarification :100:

**Pull request merged**